encord/objects/types.py

```diff
@@ -203,18 +205,35 @@ class ClassificationAnswer(ClassificationAnswerRequired, total=False):
     lastEditedBy: Union[str, None]
     manualAnnotation: Union[bool, None]
     reviews: list[Any]  # TODO: Remove this as its deprecated
+    spaces: Dict[str, SpaceRange]  # Only exists if item is on a space
 
 
-class ObjectAnswer(TypedDict):
+class ObjectAnswerForGeometric(TypedDict):
     objectHash: str
     classifications: List[AttributeDict]
 
 
+class SpaceRange(TypedDict):
+    range: list[list[int]]
```

Comment:
Should this `list[list[int]]` be a type alias (`SpaceRangeValue` or smth? It's repeated a bunch of times I think)